### PR TITLE
More sensible default dupFinder exclude pattern

### LINKED ISSUE #174: Remove hard coding of Cake.Example.Tests
There's a comment in toolssettings.cake that says: `TODO: Remove hard coding of Cake.Example.Tests`. This comment references the following line:
```
DupFinderExcludePattern = dupFinderExcludePattern ?? new string[] { context.MakeAbsolute(context.Environment.WorkingDirectory) + "/src/Cake.Example.Tests/*.cs" };
```

A much better default value would be:
```
var absoluteWorkingDirectory = context.MakeAbsolute(context.Environment.WorkingDirectory);
DupFinderExcludePattern = dupFinderExcludePattern ??
    new string[]
    {
        string.Format("{0}/Source/{1}.Tests/**/*.cs", absoluteWorkingDirectory, BuildParameters.Title),
        string.Format("{0}/Source/{1}/**/*.AssemblyInfo.cs", absoluteWorkingDirectory, BuildParameters.Title)
    };
```

### PR DESCRIPTION
**Pull request by Jericho**
Remove hard-coded "Cake.Example.Tests" and replace with a more sensible default dupFinder exclude pattern.

Resolves #174

### CODE REVIEW

File: Cake.Recipe/Content/toolsettings.cake [hunk 1]
```diff
@@ -29,8 +29,13 @@ public static class ToolSettings
     {
         context.Information("Setting up tools...");
 
-        // TODO: Remove hard coding of Cake.Example.Tests
-        DupFinderExcludePattern = dupFinderExcludePattern ?? new string[] { context.MakeAbsolute(context.Environment.WorkingDirectory) + "/src/Cake.Example.Tests/*.cs" };
+        var absoluteWorkingDirectory = context.MakeAbsolute(context.Environment.WorkingDirectory);
+        DupFinderExcludePattern = dupFinderExcludePattern ??
+            new string[]
+            {
+                string.Format("{0}/Source/{1}.Tests/**/*.cs", absoluteWorkingDirectory, BuildParameters.Title),
```
Comment (gep13) [inline Cake.Recipe/Content/toolsettings.cake]: @Jericho why is `Source` included here?

File: Cake.Recipe/Content/toolsettings.cake [hunk 2]
```diff
@@ -29,8 +29,13 @@ public static class ToolSettings
...
+                string.Format("{0}/Source/{1}/**/*.AssemblyInfo.cs", absoluteWorkingDirectory, BuildParameters.Title)
```
Comment (gep13) [inline Cake.Recipe/Content/toolsettings.cake]: @Jericho why is `Source` included here?

**Comment by gep13** [comment]
@Jericho in principal, I have no objection to this change, however, I don't think `Source` should be within the default exclusion pattern, as the name of this folder is configurable as well through BuildParameters.

**Comment by Jericho** [comment]
Thanks for the feedback. I improved my solution by using SolutionDirectoryPath and TestDirectoryPath.

**Comment by gep13** [comment]
@Jericho your changes have been merged, thanks for your contribution 👍

**Pull request merged**